data-model/nginx/nginx.conf

```diff
@@ -0,0 +1,17 @@
+worker_processes 4;
```

Comment:
What is the reason to put it on 4?

Reply:
All the other nginx files we have have 4 so I just went with it. 

[From this nginx blog post](https://www.nginx.com/blog/tuning-nginx/)
- 1 process can handle 512 connections by default
- recommended to have 1 process per core
- Setting this this to `auto` will achieve that (better for auto-scaling clusters)

Reply:
:+1: thanks for looking into this. Do you think we should change it in other place as well? Feel free to make a Jira issue for it

Reply:
https://naturalis.atlassian.net/browse/DD-865